microprofile.ls/com.redhat.microprofile.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-properties.json

```diff
@@ -1,14 +1,14 @@
 {
-	"Add datasource properties": {
+	"Add Panache properties": {
 		"prefix": "qds",
 		"body": [
-			"quarkus.datasource.url=${1|jdbc:mariadb://localhost:3306/mydb,jdbc:mysql://localhost:3306/test,jdbc:h2:mem:mydb,jdbc:postgresql://localhost/mydb,jdbc:sqlserver://localhost:1433;databaseName=mydb|}",
-			"quarkus.datasource.driver=${2|org.mariadb.jdbc.Driver,org.mysql.jdbc.Driver,org.h2.Driver,org.postgresql.Driver,com.microsoft.sqlserver.jdbc.SQLServerDriver|}",
+			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
+			"quarkus.datasource.db-kind=${2|h2,derby,mariadb,mssql,mysql,postgresql|}",
 			"quarkus.datasource.username=${3:developer}",
 			"quarkus.datasource.password=${4:developer}",
-			"quarkus.hibernate-orm.database.generation=${5|update,drop-and-create,create,drop,none|}"
+			"quarkus.hibernate-orm.database.generation=${5|none,drop-and-create,create,drop,update|}"
 		],
-		"description": "Configure Quarkus datasource",
+		"description": "Configure Quarkus Panache",
```

Comment:
It is a datasource not Panache. It was good before the change. Even you can use agroal alone without Panache and still need to set all these configuration parameters.

Reply:
Thank you, fixed